app/controllers/responsiblepeople/PersonNonUKPassportController.scala

```diff
@@ -55,10 +56,12 @@ class PersonNonUKPassportController @Inject()(
                          edit: Boolean, flow: Option[String] )(implicit authContext:AuthContext, request: Request[AnyContent]) = {
     (for {
       cache <- result
+      phase2Changes <- Some(appConfig.phase2ChangesToggle)
```

Comment:
Possible to just use appConfig.phase2ChangesToggle in the match statement?

Reply:
Done